.github/workflows/client-js-test.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     runs-on: ubuntu-latest
     env:
-      go-version: 1.18.3
+      go-version: 1.19.1
 
     steps:
       - uses: actions/checkout@v2
```

Comment:
```suggestion
      - uses: actions/checkout@v3
```

Reply:
This line should also be v3, too.
https://github.com/m3dev/dsps/pull/8/files#diff-35e16cd18acb537bf3ce35e5b3eb25723dc3013dc5d2cab1e400a7de8066f8f7R35